mailing_list/templates/mailing_list/_mailing_list_details.html

```diff
@@ -0,0 +1,147 @@
+          when="{'0': 'No members', 'one': '1 member', 'other': '{} members'}">
+          </ng-pluralize></a>)
+      </strong>
+    </p>
+    <p ng-class="{'caption-disabled': list.access_level == 'readonly'}">
+      <strong>
+        Mailing list address:
+      </strong>
+      {{ list.address }}
+    </p>
+  </div>
+  <div class="float-left col-50 info-btn-status">
+    <a id="email-section-{{ list.section_id }}"
+       href="mailto:{{ list.address }}"
+       class="btn"
+       {% endverbatim %}
+       {% if scope == "class" %}
+         ng-class="list.access_level == 'readonly'
+                   ? 'btn-submit-primary-disabled' : 'btn-submit-primary'"
+       {% else %}
+         ng-class="list.access_level == 'readonly'
+                   ? 'btn-submit-secondary-disabled' : 'btn-submit-secondary'"
+       {% endif %}
+       {% verbatim %}
+       ng-disabled="list.access_level == 'readonly'">
+      <i class="fa fa-envelope"></i>
+      {% endverbatim %}
+      Email
+        {% if scope == "class" %}
+          Class Mailing List
+        {% else %}
+          {% verbatim %}
+            {{ list.name }}
+          {% endverbatim %}
+        {% endif %}
+      {% verbatim %}
+    </a>
+    <p>
+      {% endverbatim %}
+      {% if scope == "class" %}
+        {% verbatim %}
+          {{ ml.accessLevelStatusClass[list.access_level] }}
+        {% endverbatim %}
+      {% else %}
+        {% verbatim %}
+          {{ ml.accessLevelStatusSection[list.access_level] }}
+        {% endverbatim %}
+      {% endif %}
+      {% verbatim %}
+      <br>
+      <a href="#"
+         data-toggle="modal"
+         data-target="#permissions-modal-{{ list.section_id }}"
+         ng-click="ml.updatedAccessLevel = list.access_level">
+        Change who can email
+        {% endverbatim %}
+        {% if scope == "class" %}
+          the class
+        {% else %}
+          this section
+        {% endif %}
+        {% verbatim %}
+      </a>
+      {% endverbatim %}
+      <img src="{% static 'images/ajax-loader-small.gif' %}"
+           ng-show="list.isUpdating"/>
+      {% verbatim %}
+      <span class="text-success feedback" ng-show="list.updated">
+        &checkmark;
+      </span>
+      <span class="text-danger feedback" ng-show="list.update_failed">
+        &cross;
+      </span>
+    </p>
+    <div id="permissions-modal-{{ list.section_id }}" class="modal fade">
+      <div class="modal-dialog">
+        <div class="modal-content">
+          <form>
+            <div class="modal-header">
+              <button type="button"
+                      class="close"
+                      data-dismiss="modal"
+                      aria-label="Close">
+                <span aria-hidden="true">&times;</span>
+              </button>
+              <h4 class="modal-title">
+                <label for="permissions-input-{{ list.section_id }}"
+                       class="ic-Label">
+                  Change permissions
```

Comment:
Sure. Uncapitalized text in the [mockup](https://projects.invisionapp.com/share/RM3DKOC35#/screens/88963970) but I think, given the button is capitalized, it makes sense.